packages/rustc/config.toml

```diff
@@ -27,7 +27,7 @@ codegen-tests = false
 [target.x86_64-unknown-linux-gnu]
 cc = "gcc"
 cxx = "g++"
-llvm-config = "/usr/bin/llvm-config"
+llvm-config = "/usr/bin/llvm-config-3.9"
```

Comment:
Forgot to mention, this didn't work for me because I'm using llvm 4.0.  Very unlikely anybody is still on 3.8 or older, think you should take this out.